operations/cpi.yml

```diff
@@ -16,4 +16,8 @@
 
 - type: replace
   path: /instance_groups/name=bosh/properties/director/enable_cpi_resize_disk?
-  value: true
\ No newline at end of file
+  value: true
+
+- type: replace
+  path: /tags?/cg_environment?
```

Comment:
Is there a reason to use `cg_environment` vs `environment`?

`environment` would be consistent with how we're tagging resources created by brokers written in Go: https://github.com/cloud-gov/go-broker-tags/blob/main/tags.go#L13